bootstrap/components/sources/kafka-source-topology-component.json

```diff
@@ -37,10 +37,11 @@
       {
         "uiName": "Reader schema version",
         "fieldName": "readerSchemaVersion",
-        "isOptional": true,
+        "isOptional": false,
```

Comment:
@priyank5485 @shahsank3t I do not think current experience is seamless, it is still kind of broken as user needs to update with reader schema version when topology is edited. 
Ideally, user should not get any validation error when importing a topology created in earlier streamline versions. It should be upgraded seamlessly by showing respective info messages. In this case, this should have been upgraded by setting the right reader version(fetch from SR by sending the current schema text and its schema name) without asking user to configure it. 
This is not really a UI issue but a general import issue which should be handled by respective sources/sinks/processors, filed issue #822 on the same.